src/hotspot/cpu/aarch64/continuationFreezeThaw_aarch64.inline.hpp

```diff
@@ -300,7 +301,9 @@ inline void ThawBase::derelativize_interpreted_frame_metadata(const frame& hf, c
 }
 
 inline void ThawBase::set_interpreter_frame_bottom(const frame& f, intptr_t* bottom) {
-  *(intptr_t**)f.addr_at(frame::interpreter_frame_locals_offset) = bottom - 1;
+  // set relativized locals
+  // this line can be changed into an assert when we have fixed the "frame padding problem"
+  *f.addr_at(frame::interpreter_frame_locals_offset) = (bottom - 1) - f.fp();
```

Comment:
Thank you for this comment.  When you file the CR for this problem, can you point to this CR?

Reply:
Created JDK-8300197 and added references in the source to i.